rest_framework/exceptions.py

```diff
@@ -235,3 +236,23 @@ def __init__(self, wait=None, detail=None, code=None):
                                      wait))))
         self.wait = wait
         super(Throttled, self).__init__(detail, code)
+
+
+def server_error(request, *args, **kwargs):
+    """
+    Generic 500 error handler.
+    """
+    data = {
+        'error': 'Server Error (500)'
```

Comment:
Would using `detail` as the key here have been more consistent with other DRF errors? I excitedly ripped out my own custom error handlers for DRF's shiny new ones only to see that the key for error details was now something unexpected.